### PR TITLE
feat: aptos data api update

### PR DESCRIPTION
**Pull request by 0xMasayoshi**
## PR-Codex overview
This PR removes unused files, updates dependencies, refactors components, and improves UI consistency in the `aptos` web app.

### Detailed summary
- Removed unused files related to pool components
- Updated dependencies including `@aptos-labs/ts-sdk`
- Refactored components for better UI consistency
- Added `Providers` component for pools filters
- Updated UI elements such as labels and styling
- Refactored SDK class for network handling

> The following files were skipped due to too many changes: `apps/web/src/app/(non-evm)/aptos/(common)/components/AddSection/AddSectionWidget.tsx`, `apps/web/src/app/(non-evm)/aptos/(common)/ui/currency/currency-input/currency-input.tsx`, `apps/web/src/app/(non-evm)/aptos/providers.tsx`, `apps/web/src/app/(non-evm)/aptos/(positions)/hero.tsx`, `apps/web/src/app/(non-evm)/aptos/pool/lib/use-top-pools.ts`, `apps/web/src/app/(non-evm)/aptos/(common)/lib/common/use-token-balances.ts`, `apps/web/src/app/(non-evm)/aptos/pool/ui/pools/tables/positions/columns.tsx`, `apps/web/src/app/(non-evm)/aptos/pool/(landing)/layout.tsx`, `apps/web/src/app/(non-evm)/aptos/layout.tsx`, `apps/web/src/app/(non-evm)/aptos/(common)/components/RemoveSection/RemoveSectionUnstake.tsx`, `apps/web/src/app/(non-evm)/aptos/(common)/components/PoolSection/PoolPosition/PoolPosition.tsx`, `apps/web/src/app/(non-evm)/aptos/explore/hero.tsx`, `apps/web/src/app/(non-evm)/aptos/(common)/components/PoolSection/PoolMyRewards.tsx`, `apps/web/src/app/(non-evm)/aptos/header.tsx`, `apps/web/src/app/(non-evm)/aptos/(common)/components/AddSection/AddSectionStake.tsx`, `apps/web/src/app/(non-evm)/aptos/(common)/components/PoolSection/PoolComposition.tsx`, `apps/web/src/app/(non-evm)/aptos/pool/ui/pools/tables/pools/pools-table.tsx`, `apps/web/src/app/(non-evm)/aptos/pool/ui/pools/tables/pools/cells/pool-name-cell.tsx`, `pnpm-lock.yaml`, `apps/web/src/app/(non-evm)/aptos/pool/[id]/page.tsx`, `apps/web/src/app/(non-evm)/aptos/(common)/components/ManageV2LiquidityCard.tsx`, `apps/web/src/app/(non-evm)/aptos/pool/ui/pools/tables/pools/columns.tsx`, `apps/web/src/app/(non-evm)/aptos/(common)/components/PoolSection/PoolHeader.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/app/(non-evm)/aptos/(positions)/providers.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use client'
+
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { PoolsFiltersProvider } from '~aptos/pool/ui/pools/filters/pool-filters-provider'
+
+export function Providers({ children }: { children: React.ReactNode }) {
+  const queryClient = new QueryClient()
```
Comment (matthewlilley) [inline apps/web/src/app/(non-evm)/aptos/(positions)/providers.tsx:7]: Might cause hydration errors without:

const queryClient = useState(() => new QueryClient())

File: apps/web/src/app/(non-evm)/aptos/(common)/lib/common/use-token-balances.ts [hunk 2]
```diff
@@ -14,21 +15,33 @@ export const tokenBalanceQueryFn = async ({
   currency,
   network,
 }: TokenBalanceQueryFn) => {
-  const response = await fetch(
-    `${chains[network].api.fetchUrlPrefix}/v1/accounts/${account}/resource/0x1::coin::CoinStore<${currency}>`,
+  const aptos = new Aptos(
```
Comment (LufyCZ) [inline apps/web/src/app/(non-evm)/aptos/(common)/lib/common/use-token-balances.ts:18]: might make sense to extract this for reusability?

File: apps/web/src/app/(non-evm)/aptos/explore/layout.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  children,
+}: { children: React.ReactNode }) {
+  return (
+    <>
+      <Container maxWidth="7xl" className="px-4 py-[9.5rem]">
+        <Hero />
+      </Container>
+      <section className="flex flex-col min-h-screen">
+        <div className="flex-1 bg-gray-50 dark:bg-white/[0.02] border-t border-accent pt-4 pb-20">
+          <Providers>{children}</Providers>
```
Comment (LufyCZ) [inline apps/web/src/app/(non-evm)/aptos/explore/layout.tsx:15]: there's already a QueryClientProvider in the level above

File: apps/web/src/app/(non-evm)/aptos/pool/[address]/page.tsx [hunk 4]
```diff
@@ -62,24 +60,23 @@ const Pool = () => {
       <div className="flex flex-col gap-6">
         <div className="grid grid-cols-1 md:grid-cols-[auto_400px] gap-6">
           <div className="flex flex-col gap-6">
-            <ManageV2LiquidityCard />
-            {pool?.id ? <PoolComposition row={pool} /> : null}
+            <ManageV2LiquidityCard address={address} />
+            <PoolComposition row={pool} />
```
Comment (LufyCZ) [inline apps/web/src/app/(non-evm)/aptos/pool/[address]/page.tsx:64]: might be nice to have loading states for these things but would be a lot of work for not a lot of gain (#TODO #FixSomeday)

File: apps/web/src/app/(non-evm)/aptos/pool/lib/use-top-pools.ts [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+      const pools = await getTopNonEvmPools({ chainId: 'aptos' })
+
+      return pools.map((pool) => ({
+        ...pool,
+        token0: tokens?.[pool.token0Address],
+        token1: tokens?.[pool.token1Address],
+      })) as TopPool[]
+    },
+    enabled: Boolean(enabled),
+    staleTime: Infinity,
```
Comment (LufyCZ) [inline apps/web/src/app/(non-evm)/aptos/pool/lib/use-top-pools.ts:58]: will `getTopNonEvmPools` ever change? if staleTime is set to Infinity...

File: apps/web/src/app/(non-evm)/aptos/explore/layout.tsx (hunk 3 above)
Comment (LufyCZ) [reply apps/web/src/app/(non-evm)/aptos/explore/layout.tsx:15]: ah guess you fixed it and I didn't have the changes locally

**Review by LufyCZ — CHANGES_REQUESTED** [review]

**Pull request merged**